debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -192,6 +192,9 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
         }
     }
 
+    public static final Field SERVER_NAME = RelationalDatabaseConnectorConfig.SERVER_NAME
```

Comment:
@gunnarmorling  Can't we have `SERVER_NAME` and `SERVER_NAME_WITH_HISTORY` fields defined in `RelationalDatabaseConnectorConfig` and refer to the one necessary by the connector?

Reply:
Yes, let's do it once all the historized configs are derived from `HistorizedRelationalDatabaseConnectorConfig` which would be the natural place for that definition.